src/CMakeLists.txt

```diff
@@ -137,7 +139,11 @@ install(FILES hrtf/mysofa.h DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
 
 if(BUILD_TESTS)
   add_executable(mysofa2json tests/sofa2json.c tests/json.c)
-  target_link_libraries(mysofa2json mysofa-static)
+  if(BUILD_STATIC_LIBS)
+    target_link_libraries(mysofa2json mysofa-static)
+  else()
```

Comment:
this is wrong. if must be if build shared

Reply:
If we are building a STATIC libs, then the current (and kept current) behavior is to build binaries with mysofa-static
So no, it should not be build shared.